src/styles/product.scss

```diff
@@ -0,0 +1,144 @@
+.product {
+  background-color: $page-backgroun-color;
```

Comment:
There's a typo in the variable name $page-backgroun-color. Should be changed to $page-background-color